### DISCUSSION TITLE
请问本仓库防止webview窗口闪烁的方法是什么？

### DISCUSSION THREAD

**Discussion by Patchethium** [zzzgydi/clash-verge#121] [Q&A] [answered]
我正在使用tauri，发现只要UI的复杂度稍高一些，在Linux上调整窗口大小的时候就会有白色闪烁现象，拖拽处也会出现白边，具体如下：

https://user-images.githubusercontent.com/74458240/177377187-ad9dc623-e8ea-4a75-bc70-3d12ee4d528e.mp4

我在tauri的官方讨论区找到了同样问题的[讨论](https://github.com/tauri-apps/tauri/discussions/2970)，维护者回复说这不是tauri的问题而是webview的，在tauri侧没有办法解决。
但我看到本仓库的应用在Linux上完全没有类似问题，因此想知道本仓库防止该问题使用的方法，谢谢。

**Comment by zzzgydi** [comment, marked as answer]
本仓库一直使用最新版本的tauri，另外我也没有针对这个问题做过什么优化和修复，所以不太清楚。我看这个讨论已经是去年11月份的了，不知道现在的tauri版本还是不是存在这个问题。

**Reply by Patchethium** [reply]
> 我看这个讨论已经是去年11月份的了，不知道现在的tauri版本还是不是存在这个问题。

现在我使用1.0的情况下还是有这个问题的，真奇怪。

> 我也没有针对这个问题做过什么优化和修复

明白了，谢谢！
